src/cascadia/TerminalApp/TerminalPage.cpp

```diff
@@ -1069,6 +1090,22 @@ namespace winrt::TerminalApp::implementation
         }
     }
 
+    winrt::Microsoft::Terminal::TerminalControl::TermControl _GetActiveControl()
```

Comment:
Why is this method called `_GetActiveControl`? It should have a doc comment, like the rest of the methods in this file.

Reply:
Honestly looks like this should just be deleted entirely...

Reply:
Thank you. Resolved.